src/tests/user.spec.js

```diff
@@ -0,0 +1,63 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '..';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const endPoint = '/api/v1';
+
+describe('POST /api/v1/auth/signup', () => {
+  it('should create a new user and return 201', (done) => {
+    const user = {
+      firstName: 'victor',
+      lastName: 'nach',
+      userName: 'lazydev',
+      email: 'myemail@gmail.com',
+      password: 'password',
+      phoneNumber: '09034343434',
+      isAdmin: false,
+    };
+    chai.request(app)
+      .post(`${endPoint}/auth/signup`)
+      .send(user)
+      .end((_err, res) => {
+        expect(res).to.have.status(201);
+        expect(res.body.status).to.be.equal(201);
+        expect(res.body).to.have.property('data');
+        expect(res.body.data).to.be.a('object');
+        expect(res.body.data).to.have.property('id');
+        expect(res.body.data.id).to.be.a('number');
+        expect(res.body.data.id % 1).to.be.equal(0);
```

Comment:
What is this testing for?

Reply:
The id is the primary key which we need to be returned. I'm testing that the response body has a field 'id' and that it is a number and also an integer. The tests pass as is.